sample_xblocks/filethumbs/filethumbs.py

```diff
@@ -59,9 +64,9 @@ def student_view(self, context=None):  # pylint: disable=W0613
         frag = Fragment(unicode(html_str))
 
         if not self.fs.exists("thumbsvotes.json"):
-            with self.fs.open("thumbsvotes.json", "wb") as f:
-                json.dump({'up': 0, 'down': 0}, f)
-                f.close()
+            with self.fs.open('thumbsvotes.json', 'wb') as file_output:
+                json.dump({'up': 0, 'down': 0}, file_output)
+                file_output.close()
```

Comment:
I disagree about pylintrc. By adding the exception, we enable developers to use more ambiguous variable names elsewhere in the code.

Also, `f` is objectively more ambiguous than `file_output`. The latter provides additional context about the file pointer and its behavior; I know what I can do with the object without inspecting its initialization.

Despite frequent arguments from developers that truncated variable names "save time typing" or some such nonsense, it's important to remember that all code will be read _many_ more times than it is written. We should _always_ favor explicit naming conventions; it's a no-lose situation.